spec/models/move_spec.rb

```diff
@@ -672,4 +672,36 @@
       end
     end
   end
+
+  describe '#all_events_for_timeline' do
+    subject(:move) { create(:move, profile: create(:profile, :with_person_escort_record)) }
```

Comment:
This isn't really the subject, so I'd prefer to see

```suggestion
    subject(:all_events_for_timeline) { move.all_events_for_timeline }

     let(:move) { create(:move, profile: create(:profile, :with_person_escort_record)) }
```